ev3dev2/console.py

```diff
@@ -38,7 +38,62 @@ def __init__(self, font="Lat15-TerminusBold24x12"):
 
         """
         self._font = None
-        self.set_font(font, False)  # don't reset the screen during construction
+        self._columns = 0
+        self._rows = 0
+        self._echo = False
+        self._cursor = False
+        self.set_font(font, reset_console=False)  # don't reset the screen during construction
+        self.cursor = False
+        self.echo = False
+
+    @property
+    def columns(self):
+        """
+        Return (int) number of columns on the EV3 LCD console supported by the current font.
+        """
+        return self._columns
+
+    @property
+    def rows(self):
+        """
+        Return (int) number of rows on the EV3 LCD console supported by the current font.
+        """
+        return self._rows
+
+    @property
+    def echo(self):
+        """
+        Return (bool) whether the console echo mode is enabled.
+        """
+        return self._echo
+
+    @rows.setter
+    def echo(self, value):
```

Comment:
`rows.setter`? Shouldn't that be `echo` (here and below)?

Reply:
Good catch! Yes. Fix is imminent. Looks like a copy pasta mistake.